### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by hima-v** [ModECI/MDF#507] [General]
Hello everyone, my name is Hima Verma. I am an Outreachy Applicant for 2024 batch. I am looking forward to contribute and collaborate under the mentor's (Padraig Gleeson) guidance. I especially enjoyed my course in Probabilistic Graphical Models, and I find the project's initiative and goal really innovative and crucial from a perspective of advancing interdisciplinary collaboration across various domains.

**Comment by pgleeson** [comment]
Welcome @hima-v! See https://github.com/ModECI/MDF/discussions/509 for more details.

**Comment by codinghappiness-web** [comment]
Hello everyone, my name is Happiness, I am an Outreachy applicant and I am excited to be contributing to this project
